.github/workflows/codecov.yml

```
@@ -0,0 +1,36 @@
+  id-token: write
+
+jobs:
+  published-package-coverage:
+    name: Published Package Coverage
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Install Dependencies
+        uses: ./.github/workflows/actions/install-dependencies
```

actionlint (1.7.12)

[error] could not parse action metadata in "/home/jailuser/git/.github/workflows/actions/install-dependencies": line 5: unexpected key "type" for definition of input "version"

(action)